packages/adblocker/test/utils.ts

```diff
@@ -43,11 +43,62 @@ export function getNaughtyStrings(): string[] {
   return fs.readFileSync(path.resolve(__dirname, 'data', 'blns.txt'), 'utf-8').split('\n');
 }
 
-export function getRawTrackerDB(): any {
-  const trackerdb = JSON.parse(
-    zlib.unzipSync(
-      fs.readFileSync(path.resolve(__dirname, 'data', 'trackerdb_20221213.json.gz'))
-    ).toString('utf-8'),
+export type NullableFields<T extends Record<string, unknown>, K extends keyof T = keyof T> = {
+  [key in K]: T[K] | null;
+};
+
+export type TrackerDBCategory = {
+  name: string;
+  color: string;
+  description: string;
+};
+
+export type TrackerDBPattern = {
+  name: string;
+} & NullableFields<{
+  category: string;
+  organization: string;
+  alias: any;
+  website_url: any;
+  ghostery_id: string;
+  domains: Array<string>;
+  filters: Array<string>;
+}>;
+
+export type TrackerDBOraganization = {
+  name: string;
+} & NullableFields<{
+  description: string;
+  website_url: string;
+  country: string;
+  privacy_policy_url: string;
+  privacy_contact: string;
+  ghostery_id: string;
+}>;
```

Comment:
Why this is here?

Reply:
I didn't find a typescript type exports from trackerdb. I think it's best to keep the correct types for the potential future use.

Reply:
cannot we extract them from source in another way? duplication does not seem as a good solution

Reply:
Since we need to ungzip the data, typescript won't able to see the types.